pkg/yaml/yaml.go

```diff
@@ -66,11 +62,10 @@ func NewDecoder(data []byte, strict bool) func(interface{}) error {
 }
 
 func Marshal(v interface{}) ([]byte, error) {
-	if GoccyGoYaml {
+	if runtime.GoccyGoYaml {
 		var b bytes.Buffer
 		yamlEncoder := yaml.NewEncoder(
 			&b,
-			yaml.IndentSequence(true),
```

Comment:
why do remove this option?

Reply:
It turned out that this changed how Helmfile formatted YAML. The previous behavior with gopkg.in/yaml.v2 seems to correspond to goccy/go-yaml WITHOUT the IndentSequence option.